### ISSUE TITLE
High contrast color support

### ISSUE THREAD

**Issue by dgomezc** [Microsoft/WindowsTemplateStudio#1887]
Some color combinations are not correct in high contrast mode:

- MVVMLight icon not visible in high contrast. (Fixed in #1695).

![image](https://user-images.githubusercontent.com/12777039/36159040-0e9d5c24-10de-11e8-8141-64833b3952a1.png)

- When any cards is focused or selected, the icon color is very similar to card background. The details hyperlink does not read correctly.

![image](https://user-images.githubusercontent.com/12777039/36159172-5c32e99a-10de-11e8-8489-680d35781e86.png)

- In changes summary, the file title in code control view does not read correctly because the background color is similar to text color.

![image](https://user-images.githubusercontent.com/12777039/36159329-b9ac3ae0-10de-11e8-9492-7cc71cebb632.png)

- With high contrast theme 2, the selected step item in left panel does no read correctly.

![image](https://user-images.githubusercontent.com/12777039/36159517-358d6a94-10df-11e8-935c-2dace2a398d5.png)

**Comment by crutkas** [comment]
why doesn't the icon take the color of the text?

**Comment by sibille** [comment]
@crutkas, the icon has it's own color category. What's missing is choosing the correct color for this in high contrast (it's currently set to grey for all 4 themes)

**Comment by crutkas** [comment]
have it match the title.

**Comment by mvegaca** [comment]
High Contrast color updates in a [separated branch](https://github.com/mvegaca/WindowsTemplateStudio/tree/Issue1903-colors).
CardBorderSelect has the same value that CardBorderNormal in High Contrast

CardBorderSelected
Light
#007ACC
Blue
#E5C365
Dark
#007ACC
High Contrast
Highlight

![image](https://user-images.githubusercontent.com/5556784/36733539-356cdaac-1bd1-11e8-9810-3c80cd727892.png)

ChangesSummaryDetailFileHeaderBackground has wrong value on dark and high contrast.

ChangesSummaryDetailFileHeaderBackground
Light
3EFEFF2
Dark
#2D2D30
High Contrast
Highlight

Updated values

Dark
![image](https://user-images.githubusercontent.com/5556784/36733679-98076fd8-1bd1-11e8-8a58-3cbd253e6a77.png)

High Contrast
![image](https://user-images.githubusercontent.com/5556784/36733723-beb5e89e-1bd1-11e8-97f4-f6c5cb088422.png)

**Comment by mvegaca** [comment]
These are the updated color in High Contrast on WTS Color Category.


Windows Template Studio Color | High Contrast Value
-- | --
CardFooterText | GrayText
ChangesSummaryDetailFileHeaderBackground | Highlight
CardBorderSelected | Highlight
NewItemFileStatusNewFile | Highlight
NewItemFileStatusConflictingFile | Highlight
NewItemFileStatusConflictingStylesFile | Highlight
NewItemFileStatusWarningFile | Highlight
NewItemFileStatusUnchangedFile | Highlight
CardIcon | ControlText
TemplateInfoPageDescription | ControlText


- CardFootertext is used on template added count on the Added label.
- CardBorderSelected is used to show what card is selected on Project Type, Framework or Template (right click).
- CardIcon is the card icon of each template, framework and project type.
- TemplateInfoPageDescription is used on Template info page (clicking on card details link).
- ChangesSummaryDetailFileHeaderBackground is used on right click changes summary page on file header background.
- FileStatus colors are used on changes summary page to represent the kind of file on the generation result. We were thinking that there are not enough colors in high contrast to represent that kind of files and we decided to set all circles as Highlighted in High Contrast

ScreenShots on High Contrast mode:

![image](https://user-images.githubusercontent.com/5556784/36792445-e7598eee-1c9a-11e8-97a8-9637f6dd9901.png)

![image](https://user-images.githubusercontent.com/5556784/36792467-f45716f2-1c9a-11e8-822d-6a9c6d3a983e.png)

![image](https://user-images.githubusercontent.com/5556784/36792523-17693bc0-1c9b-11e8-99da-4e3714573fe1.png)

![image](https://user-images.githubusercontent.com/5556784/36792570-31739c4a-1c9b-11e8-8b44-11c1e4bfae49.png)

![image](https://user-images.githubusercontent.com/5556784/36792591-3ca1763c-1c9b-11e8-8a05-44a5e5247d81.png)

**Comment by mvegaca** [comment]
Fixed
Templates version: 0.17.18060.3
Wizard version: 0.17.18060.3
